.cmake-format.py

```diff
@@ -0,0 +1,257 @@
+# ----------------------------------
+# Options affecting listfile parsing
+# ----------------------------------
+with section("parse"):
+
+  # Specify structure for custom cmake functions
+  additional_commands = {
+    'install': {
+      'flags': [],
+      'kwargs': {
+        'TARGETS': '*',
+        'DIRECTORY': 3,
+        'REGEX': 2,
+        'BUNDLE': 2,
+        'RUNTIME': 2,
+        'ARCHIVE': 2,
+        'LIBRARY': 2,
+        'INCLUDES': 2,
+      },
+    },
+  }
+
+  # Override configurations per-command where available
+  override_spec = {}
+
+  # Specify variable tags.
+  vartags = []
+
+  # Specify property tags.
+  proptags = []
+
+# -----------------------------
+# Options affecting formatting.
+# -----------------------------
+with section("format"):
+
+  # Disable formatting entirely, making cmake-format a no-op
+  disable = False
+
+  # How wide to allow formatted cmake files
+  line_width = 80
+
+  # How many spaces to tab for indent
+  tab_size = 2
```

Comment:
If you want to change the indentation, you should update [.editorconfig](https://github.com/LibrePCB/LibrePCB/blob/dc0e9c0a6efb668ce36430fd03a9fd824f68da8d/.editorconfig#L13) as well.

Reply:
Good point, thanks! -> Updated `.editorconfig`.